code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -751,4 +751,23 @@
 		M.adjustStaminaLoss(10)
 	if(prob(30))
 		M << "You should sit down and take a rest..."
-	..()
\ No newline at end of file
+	..()
+
+/datum/reagent/toxin/seawater //You get this when you learn that breathing ocean water is a bad idea.
+	name = "Seawater"
+	id = "seawater"
+	description = "Ocean water. Probably not good for breathing."
+	toxpwr = 0
+	metabolization_rate = 1 * REAGENTS_METABOLISM
```

Comment:
technically not an issue, byond's (very limited) constant folding should see this, figure out 1* REAGENTS_METABOLISM and then always use the result value instead of recalculating (yes that does mean that it actually does the operation once, Byond's constant folding isn't great)

but yes it's stupid and should be fixed by the human developer here.